tcms/testplans/tests/test_new_plan.py

```diff
@@ -76,36 +118,117 @@ def test_plan_create_new_inactive(self):
         self._test_plan_create_new(is_active=False)
 
     def _test_plan_create_new(self, is_active):
-        self.request['is_active'] = is_active
+        self.request["is_active"] = is_active
 
-        response = self.client.post(self.location, self.request)
+        response = self.client.post(self.url, self.request)
         self.assertEqual(response.status_code, HTTPStatus.FOUND)
 
         plan = TestPlan.objects.get(
-            name=self.plan_name,
+            name=self.request["name"],
             is_active=is_active,
+            product=self.request["product"],
         )
-        self.assertEqual(plan.author, self.user)
+        self.assertEqual(plan.author, self.tester)
         self.assertEqual(plan.product, self.product)
         self.assertEqual(plan.product_version, self.product_version)
-        self.assertEqual(plan.type, self.plan_type)
+        self.assertEqual(plan.type, self.test_plan_type)
```

Comment:
Keep the attribute names consistent to minimize unnecessary changes.

Reply:
yeah, sorry about that.

Reply:
changing `user` to `tester` is because of inheriting from `PermissionsTestCase`